Procfile.dev

```diff
@@ -1,3 +1,3 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
```

Comment:
@jamiefolsom - I changed this to use `puma`, as `unicorn` didn't appear to be included in the `Gemfile`.